pkg/metrics/metrics.jsx

```diff
@@ -520,21 +525,46 @@
                         </div>
                     </Tooltip>);
             }
-            const value_text = <TableText wrapModifier="nowrap">{value}</TableText>;
+            const values_text = values.map(value => {
+                return ({ title: <TableText wrapModifier="nowrap">{value}</TableText> });
+            });
             return {
-                cells: [{ title: name_text }, { title: value_text }]
+                cells: [{ title: name_text }, ...values_text]
             };
         }
 
         // top 5 CPU and memory consuming systemd units
         const topServicesCPU = n_biggest(this.cgroupCPUNames, this.samples[9], 5);
         newState.topServicesCPU = topServicesCPU.map(
-            ([key, value, is_user, is_container, userid]) => cgroupRow(key, Number(value / 10 / numCpu).toFixed(1), is_user, is_container, userid) // usec/s → percent
+            ([key, value, is_user, is_container, userid]) => cgroupRow(key, is_user, is_container, userid, Number(value / 10 / numCpu).toFixed(1)) // usec/s → percent
         );
 
         const topServicesMemory = n_biggest(this.cgroupMemoryNames, this.samples[10], 5);
         newState.topServicesMemory = topServicesMemory.map(
-            ([key, value, is_user, is_container, userid]) => cgroupRow(key, cockpit.format_bytes(value), is_user, is_container, userid)
+            ([key, value, is_user, is_container, userid]) => cgroupRow(key, is_user, is_container, userid, cockpit.format_bytes(value))
+        );
+
+        const cgroupsIOsum = this.samples[16].map((elm, i) => elm + this.samples[17][i]);
+        // get top 5 services based od cumulative IO
+        const top5IO = n_biggest(this.cgroupDiskNames, cgroupsIOsum, 5);
+        // replace sum value in top5IO with separete read, write values
+        const topServicesDiskIO = this.cgroupDiskNames.map((cgroupName, i) => [cgroupName, this.samples[16][i], this.samples[17][i]])
+                .reduce((pV, cgroup) => {
+                    for (let i = 0; i < top5IO.length; i++) {
+                        // is container || cgroup name match
+                        if ((cgroup[0].match(podmanCgroupRe) && cgroup[0].includes(top5IO[i][0])) ||
+                            (top5IO[i][4] === cgroup[0])) {
+                            // push [key, is_user, is_container, userid, read, write]
+                            pV[i] = [top5IO[i][0], top5IO[i][2], top5IO[i][3], top5IO[i][4], cgroup[1], cgroup[2]];
+                            return pV;
+                        }
+                    }
+
+                    return pV;
+                }, []);
+
+        newState.topServicesDiskIO = topServicesDiskIO.map(
+            ([key, is_user, is_container, userid, read, write]) => cgroupRow(key, is_user, is_container, userid, cockpit.format_bytes_per_sec(read), cockpit.format_bytes_per_sec(write))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18070-20221220-124215-d2339978-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)